### PR TITLE
Add heuristique link

### LINKED ISSUE #183: Intégration d'un lien vers l'Heuristique dans la section ETS
** Feature asked **
De la même manière que l'application propose des cards pour Moodle, MonETS, Bibliotech, etc... l'Heuristique souhaite être ajouté dans la liste. 

Leur logo a été transmis, pret pour intégration

### PR DESCRIPTION
**Pull request by apomalyn**
This will close #183 
Résultat validé avec eux.

### CODE REVIEW

File: android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -194,4 +194,6 @@
     </string-array>
     <string name="label_miscellaneous_pref_category">Divers</string>
     <string name="label_shake_bug_reporter_invocation_method_pref">Secouer pour rapporter un bogue</string>
+    <string name="uri_heuristique" translatable="false">http://lheuristique.ca/</string>
```
Comment (Sonphil) [inline android/app/src/main/res/values/strings.xml]: Can you move the url to this [const.xml](https://github.com/ApplETS/Notre-Dame/blob/master/android/app/src/main/res/values/const.xml)?

File: android/app/src/main/res/values-en/strings.xml [hunk 2]
```diff
@@ -195,4 +195,5 @@
     </string-array>
     <string name="label_miscellaneous_pref_category">Miscellaneous</string>
     <string name="label_shake_bug_reporter_invocation_method_pref">Shake to report a bug</string>
+    <string name="title_heuristique" translatable="false">L\'heuristique</string>
```
Comment (Sonphil) [inline android/app/src/main/res/values-en/strings.xml]: Can you move the title below line 100 (in the `<!-- ÉTS -->` block)?

File: android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (apomalyn) [reply android/app/src/main/res/values/strings.xml]: Done !

File: android/app/src/main/res/values-en/strings.xml (hunk 2 above)
Comment (apomalyn) [reply android/app/src/main/res/values-en/strings.xml]: Yes of course ! Sorry I used the translations editor 😅

**Pull request merged**